### ISSUE TITLE
Passive event listeners cannot be removed

### ISSUE THREAD

**Issue by FarSeeing** [opbeat/opbeat-angular#10]
Hi. I do understand it's not opbeat issue but you're using forked version of zone.js that cannot remove passive event listeners (to be more exact, any event listener that has object passed as options) so they remain forever.

**Comment by hmdhk** [comment]
Hi @FarSeeing ,

Thanks for reporting the issue.
Would you please provide the specific code that had the issue? it would make reproducing the issue  easier.

Cheers

**Comment by FarSeeing** [comment]
Sorry, had to do that earlier.
Just a simple code according to https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener:
```javascript
element.addEventListener('foobar', someHandler, {passive: true}); // That works well
element.removeEventListener('foobar', someHandler, {passive: true}); // That doesn't
```
The listener is not removed because forked zone.js treats last (third) parameter to addEventListener/removeEventListener as boolean `useCapture` and tries to compare objects.

**Comment by hmdhk** [comment]
Thank you for the extra info.
Let me take a look.

**Comment by hmdhk** [comment]
@FarSeeing ,

I have released `opbeat-angular@3.15.0` which fixes this issue.
Let me know how it works for you.

Cheers
